testing/async-profiler/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Andrey Pustovetov <andrey.pustovetov@gmail.com>
+# Maintainer: Andrey Pustovetov <andrey.pustovetov@gmail.com>
+pkgname=async-profiler
+pkgver=1.5
+pkgrel=0
+pkgdesc="Sampling CPU and HEAP profiler for Java featuring AsyncGetCallTrace + perf_events"
+url="https://github.com/jvm-profiling-tools/async-profiler"
+arch="all"
+depends="bash"
+license="Apache-2.0"
+options="!checkroot" # due to fakeroot the test suite fails
+makedepends="linux-headers openjdk8"
+checkdepends="bash openjdk8-dbg"
+source="$pkgname-$pkgver.tar.gz::https://github.com/jvm-profiling-tools/async-profiler/archive/v$pkgver.tar.gz
+	thread-smoke-test_should_do_some_real_work.patch
+	"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	make
+}
+
+check() {
+	cd "$builddir"
+	sudo bash -c 'echo 1 > /proc/sys/kernel/perf_event_paranoid'
```

Comment:
no idea how builders will work, needs someone more familiar with it /cc @ncopa

Reply:
This MR is still relevant.

Reply:
Not sure if this is acceptable, but if it would, you could just use `sh -c`, instead of `bash -c` here.

Reply:
I surely hope that the builders do not allow sudo without authentication.

Reply:
I excluded tests depended to modify `/proc/sys/kernel/perf_event_paranoid`.